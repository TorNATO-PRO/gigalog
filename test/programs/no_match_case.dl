parent(john, mary).

sibling(X, Y) :- parent(Z, X), parent(Z, Y).
