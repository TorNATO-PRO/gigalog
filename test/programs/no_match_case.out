parent(john,mary)
sibling(mary,mary)
